src/main/java/calculator/StringAddCalculator.java

```diff
@@ -0,0 +1,68 @@
+package calculator;
+
+import calculator.exception.StringAddException;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static java.lang.Character.*;
+
+
+public class StringAddCalculator {
+
+    public static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
```

Comment:
```suggestion
    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
```
상수화 👍 public하게 오픈하지 않아도 되어보이네요 :)